src/stories/Blocks/header/Header.tsx

```diff
@@ -101,20 +99,30 @@ export const Header = (props: HeaderProps) => {
           </nav>
           <div>
             <div className="header__menu-search">
-              <form className="header__menu-search-form">
```

Comment:
If `.header__menu-search-form` is no longer used in any part of the project, it should be deleted from `src/stories/Blocks/header/header.scss`